evalml/tests/pipeline_tests/test_time_series_pipeline.py

```diff
@@ -1068,20 +1088,32 @@ def test_binary_predict_pipeline_objective_mismatch(
         ProblemTypes.TIME_SERIES_REGRESSION,
     ],
 )
-def test_time_series_pipeline_not_fitted_error(
+def test_time_series_pipeline_no_ytrain_predict_error(
```

Comment:
If I'm reading these correctly in that all the pipeline setup is identical, I don't think we need y_train and X_train to be separate. We can just run one raises test after the other to cut down on repeated code.